PCbuild/python.props

```diff
@@ -87,6 +87,7 @@
     <nasmDir Condition="$(nasmDir) == ''">$(ExternalsDir)\nasm-2.11.06\</nasmDir>
     <zlibDir Condition="$(zlibDir) == ''">$(ExternalsDir)\zlib-1.3.1\</zlibDir>
     <zlibNgDir Condition="$(zlibNgDir) == ''">$(ExternalsDir)\zlib-ng-2.2.4\</zlibNgDir>
+    <zstdDir Condition="$(zstdDir) == ''">$(ExternalsDir)\zstd-1.5.7\lib\</zstdDir>
```

Comment:
Sorry to be a pain, just spotted this. Can we move the `lib\` from here to everywhere that references this? (Or another variable that is based on this one).

These variables are quasi-public API, in that they're deliberately overridable by builders to use alternate dependencies. We should keep them all following the same pattern - that is, pointing to the root of the sources.

Reply:
Done: [`1ae3a76` (#133366)](https://github.com/python/cpython/pull/133366/commits/1ae3a76e4827f2c8c005f92697683ca87d97fe5e)